config/roots.php

```diff
@@ -5,8 +5,16 @@
     'kirby' => function (array $roots) {
         return realpath(__DIR__ . '/../');
     },
+
+    // i18n
+    'i18n' => function (array $roots) {
+        return $roots['kirby'] . '/i18n';
+    },
     'translations' => function (array $roots) {
-        return $roots['kirby'] . '/translations';
+        return $roots['i18n'] . '/translations';
+    },
+    'i18n:rules' => function (array $roots) {
```

Comment:
I had that, but then I thought that it's not really worth the breaking change.

Reply:
I think in the long term it will be better to keep this consistent. It's not really a breaking change if the function that returns the `i18n:translations` root uses the `translations` root internally for now but if we then deprecate the old root and remove it in 3.3.0 for example.

Reply:
Probably no one overwrites that root anyway. :)

Reply:
Good idea, make `i18n:translation` the main one and `translations` an alias for now